otx/algorithms/common/adapters/mmcv/hooks/lr_updater_hook.py

```diff
@@ -1,4 +1,4 @@
-"""NNCF task related hooks."""
```

Comment:
@cih9088 I moved `BaseLrUpdaterHook` and `CustomLrUpdaterHook` from mmseg.nncf.hooks to mmcv.hooks, it's only used for self-supervised learning for semantic segmentation. Could you verify whether is it not used for NNCF?

Reply:
I can't trace why they are under `nncf`. If tests are fine, no objection.